### PR TITLE
Review of Repository | Changes to be made

### LINKED ISSUE #9: Review repository before adding to pythonindia org
Changes need to be made before introducing the inpycon-twitterbot to the pythonindia org. 
Please refer to #10

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Fixes #9 
Due to a mishap, #7 was merged, when it was still WIP. I have included a summary of changes there only.  Please refer to #7  

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-consumer_key = "Your Consumer Key here"
-consumer_secret = "Your Consumer Secret here"
-access_token = "Your Access Token here"
-access_token_secret = "Your Access Secret here"
-volunteertwitterhandle = "Your Twitter handle"
+consumer_key = "c9pXLroOFv2y9wr4oLZh6Wlov"
```
Comment (inovizz) [inline config.py]: Please don't commit the credentials here.

File: main.py [hunk 2]
```diff
@@ -1,16 +1,9 @@
-#Required imports
-
-import tweepy #for twitter API
-import time   #for giving 15 seconds break between tweets
+import tweepy
```
Comment (inovizz) [inline main.py]: Imports in incorrect order, standard library comes before third party imports.

File: main.py [hunk 3]
```diff
@@ -20,13 +13,12 @@
...
     for searchterm in searchterms:
-        
         for tweet in tweepy.Cursor(api.search, q=searchterm).items(10):
             if tweet in tweets:
                 pass
             else:
-                tweets.append(tweet.text)  # AAdding the tweet to db
-                if tweet.author.id not in names:  # Checking if the same person tweeted more than once
+                tweets.append(tweet.text)  # Adding tweet to db
+                if tweet.author.id not in names:  # Checking if same user tweeted more than once
```
Comment (inovizz) [inline main.py]: let's follow pep8 and limit the line length to 79 characters.

Also, I'm sure pylint score for this script would be less than 5, lets make 9+.

**Review by inovizz — CHANGES_REQUESTED** [review]

**Comment by inovizz** [comment]
I added some comments on your PR @vipulgupta2048 but looks like there are other issues as well. I'll list them here, probably you or @Rohithgilla12 or @a2un  can work on it.

- code is not pep8 compatible, please clean up. Use pylint of pep8 :+1: 
- namesperson is unused, can we remove it? :+1: 
- the bot is not liking the tweets, lets add code so that bot likes the tweet as well.
- The retweet part of the code doesn't look good to me, can me do retweet with comment as referenced here - https://stackoverflow.com/a/33620770/1356851 (In comment we need to mention #PyConIndia2018 and #PyConIndia no need to add RT and twitter handle again for now)

File: main.py (hunk 3 above)
Comment (vipulgupta2048) [reply main.py]: If you refer to #7, I tried to implement flake8. But couldn't 
Will try again.

Comment (inovizz) [reply main.py]: Sure.

**Comment by vipulgupta2048** [comment]
Thanks for the review @inovizz, I will add the changes you requested

File: config.py (hunk 1 above)
Comment (vipulgupta2048) [reply config.py]: Sorry, I accidentally might have committed that because of the bad merge.

File: main.py (hunk 2 above)
Comment (vipulgupta2048) [reply main.py]: Yep, a good catch. Implemented.

**Comment by Rohithgilla12** [comment]
Thanks for the review @inovizz 
Yeah we can delete the namesperson thing,
in Pycharm it shown me as PEP 8 compatilble only, anyways i will check once,
Comment feature I guess, cannot be done, but anyways we will give it a shot.
Thanks @vipulgupta2048  for your help

**Comment by Rohithgilla12** [comment]
i have updated the main.py file and now the pylint score is 10/10

**Comment by vipulgupta2048** [comment]
@Rohithgilla12 **Please do not rush forward with the changes as and when required.** 

When a PR is open, then changing any other files related to that PR creates conflicts in files I am already working on. This leads to alot of problems, not to mention time being wasted in figuring out what went wrong. I have already implemented pylint in my local and was going to push it. Please be extremely careful and not do anything rashly from now on. Ask other collaborators before taking any actions if you have any doubts. I did advise you strictly to not interfere earlier as you merged the WIP #7 into the master branch, trashing all my previous work and commits. I do understand you are new to open-source and still learning. But would appreciate if you follow the instructions told to you.

**Comment by Rohithgilla12** [comment]
Oh, I didn't know that 
Sorry I'll not touch anything now 😅
I'm really sorry for being like that :( 
From now I'll ask anything before doing

**Comment by vipulgupta2048** [comment]
@inovizz @Rohithgilla12 @ananyo2012 Please review.

**Comment by inovizz** [comment]
LGTM :+1:

**Pull request merged**